lte/protos/session_manager.proto

```diff
@@ -77,13 +77,29 @@ message RatSpecificContext {
   }
 }
 
+message TeidSet {
+  bytes teid = 1;
+  bytes end_ipv4_addr = 2;
+}
+
+enum NotifyUeEvents {
+   PDU_SESSION_INACTIVE_NOTIFY = 0; //AMF <=> SMF
+   UE_IDLE_MODE_NOTIFY = 1;         //AMF  => SMF
+   UE_PAGING_NOTIFY = 2;            //SMF  => AMF
+   UE_PERIODIC_REG_ACTIVE_MODE_NOTIFY = 3; //AMF  => SMF
+   PDU_SESSION_STATE_NOTIFY = 4;  //SMF => AMF
+   UE_SERVICE_REQUEST_ON_PAGING = 5; //AMF -> SMF
+}
+
 message RatSpecificNotification {
-  bytes pdu_session_id = 1;            //One byte
+  uint32 pdu_session_id = 1;            //One byte
   RequestType request_type = 2;
   AccessType access_type = 3;
```

Comment:
How is this AccessType different than RATType?

Reply:
We realized,  I removed  the defination and usage.